### ISSUE TITLE
LOTR mod update(continuously if possible)

### ISSUE THREAD

**Issue by nickseo7** [GreatOrator/TolkienCraftII#138]
## Description
LOTR is still being updated in 1.7.10, and now it reached v34.3. As it's one of the most important mods in this pack, the update should be worthy.
## Base information
* Minecraft version: 1.7.10
* Minecraft Forge version: 10.13.4.1566
* TolkienCraft II version: 2.9.5

## Crash report
```
CRASH REPORT HERE
```

**Comment by GreatOrator** [comment]
I can look in to it, though to be fair the last update I did was over 2 years ago and have since been working on tolkiencraft 3 for 1.10.x.

**Comment by GreatOrator** [comment]
So I looked it over, and sadly I will be unable to do any further updates.  The pack was built when FTB was still very much a thing, so the mods in the pack took no consideration of what permissions curse had or otherwise as it was done and listed using a tool FTB had.  I no longer have access to that submission tool so would have no way to push out a pack update any longer.

When I checked the pack on the launcher there are about 20 mods I would either have to locate permissions for curse or redo the mod list to make it so I could resubmit to curse.

I would simply suggest at this point updating the lotr mod locally (backup the map first!) load the pack, if all goes well for the map, upload the map to your server and update the lotr mod there as well.
